packages/vue/src/dotlottie.ts

```diff
@@ -32,11 +35,24 @@ export const DotLottieVue = defineComponent({
     speed: { type: Number, required: false },
     src: { type: String, required: false },
     useFrameInterpolation: { type: Boolean, required: false },
+    marker: { type: String, required: false },
+    autoResizeCanvas: { type: Boolean, required: false, default: true },
+    playOnHover: { type: Boolean, required: false },
```

Comment:
how about `themeId` & `animationId` ? any other props left not implemented ?

Reply:
Sorry, was following an old commit of react player. Added the latest updates. animationId, themeId and themeData